src/util/instanceConfigInheritance.tsx

```diff
@@ -107,11 +24,26 @@ export const figureInheritedValue = (
     }
   }
 
-  if (values.type === "instance") {
-    return getInstanceDefaults(values, formField);
+  if (configKey === "limits.cpu" && values.type === "instance") {
+    if (values.instanceType === "container") {
+      return ["-", "LXD (container)"];
+    } else {
+      return ["1", "LXD (VM)"];
+    }
+  }
+
+  if (configKey === "limits.memory" && values.type === "instance") {
+    if (values.instanceType === "container") {
+      return ["-", "LXD (container)"];
+    } else {
+      return ["1GB", "LXD (VM)"];
```

Comment:
Where do these "magic values" for VMs come from?

Reply:
Cpu and memory limit are special. The defaultDesc from the config api has a value of ``1Gib` (VMs)`. We can't parse it, so we have those two still hard coded in the UI with some logic around. I think it's reasonable to have the two exceptions.

Reply:
Fair enough.

Reply:
The problem is that if they change on core side in the future, and we don't know/don't realise, we'll have an inconsistent value here.